os/migrating-to-clcs.md

```diff
@@ -0,0 +1,373 @@
+# Migrating from Cloud Configs to Container Linux Configs
+
+Up until $DATE, the recommended way to provision a Container Linux machine was with a cloud config. This was a YAML file specifying things like systemd units to run, users that should exist, and files that should be written. This file would be given to a Container Linux machine, and saved on disk. Then a utility called cloudinit running in a systemd unit would read this file, look at the system state, and make necessary changes on every boot.
+
+Now a new type of config file exists, called a Container Linux Config (or CLC for short). This new config file is now the recommended way to provision a Container Linux machine.
+
+This document details how to convert an existing cloud config into a CLC. For information on what to do with a CLC, take a look at the [Config Transpiler's Getting Started page][ct-getting-started].
```

Comment:
Expand CLC and put a short description of CT in here and link to the "getting started" doc.